app/controllers/admins/mou_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Admins::MouController < ApplicationController
```

Comment:
Can I do things with this controller if I'm not a super-admin?

Reply:
👍